spec/rails_helper.rb

```diff
@@ -86,6 +86,12 @@
     Flipper.disable(:cw_forms)
   end
 
+  config.around(:each, :household_section_flag) do |example|
```

Comment:
a question on this - it looks like we turn the flag on for each test that is run, then back off. So I'm confused why `applicant_form_spec.rb:40` is passing 🤔

Reply:
@HettieS we only turn the `household_section` feature flag on for tests that have the `household_section_flag` _test_ flag. `household_section_flag` is _not_ specified for `applicant_form_spec.rb:40` (either the "it" block itself, or a parent "context" or "describe" block). Whereas it _is_ set on applicant_form_spec.rb:44